ra/ra_test.go

```diff
@@ -204,6 +197,13 @@ var (
 	log = blog.UseMock()
 )
 
+var AuthzRequest = &rapb.NewAuthorizationRequest{
+	Authz: &corepb.Authorization{
+		Identifier: "not-example.com",
+	},
+	RegID: Registration.ID,
+}
```

Comment:
I believe this can be in the `var` block like before; it probably just needs to be moved down below the declaration of `Registration`.

Reply:
done